manifest/fixtures/v2s2.manifest.json

```diff
@@ -1,26 +0,0 @@
-{
```

Comment:
Why is this being removed? The tests depend on it.

On the contrary, we should have an OCI fixture and test for `GuessMIMEType`.